side-navigation.yaml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
Nit: do we need these new characters at the top of the file?

```suggestion
```

Reply:
@advl These characters are back, I wonder if you have some autoformatter for YAML that is adding these?